Automattic/wp-calypso
# Reader: remove reader/stream-header component

The tag stream and list stream previously used the stream header component, but this is no longer required after the launch of the Refresh.

### To test

Try a tag stream, e.g.

http://calypso.localhost:3000/tag/hongkong

...and a list stream, e.g.

http://calypso.localhost:3000/read/list/blowery/comics

and make sure the stream headers look correct.

<img width="867" alt="screen shot 2016-12-19 at 16 33 27" src="https://cloud.githubusercontent.com/assets/17325/21300445/eb7e4ecc-c608-11e6-872b-38f1ae669d3b.png">

<img width="900" alt="screen shot 2016-12-19 at 16 31 34" src="https://cloud.githubusercontent.com/assets/17325/21300416/a3ce507c-c608-11e6-8f45-56f67aec9d58.png">
